docs/en/ingest-management/elastic-agent/elastic-agent-unprivileged-mode.asciidoc

```diff
@@ -36,50 +36,89 @@ Doing so may result in <<agent-sudo-error,an error>> due to the agent not having
+In addition to the <<unprivileged-integrations,integrations that are not available>> when {agent} is run in unpriviledged mode, certain data streams are also not available. The following tables show, for different operating systems, the impact when the agent does not have full administrative privileges. In most cases the limitations can be mediated by granting permissions for a user or group to the files indicated.
 
 .macOS
 [options,header]
 |===
-|Command |Behavior in unprivileged mode |Error message
+|Action |Behavior in unprivileged mode |Resolution
 
-|<<elastic-agent-inspect-command,`inspect` command>>
-|Returns only...
-|`Exmaple error message...`
+|Run {agent} with the System integration
+|Log file error: `Unexpected file opening error: Failed opening /var/log/system.log: open /var/log/system.log: permission denied`.
+|Give read permission to the `elastic-agent` group for the `/var/log/system.log` file to fix this error.
+
+|Run {agent} with the System integration
+|On the `[Logs System] Syslog` dashboard, the `Syslog events by hostname`, `Syslog hostnames and processes` and `Syslog logs` visualizations are are missing data.
+|Give read permission to the `elastic-agent` group for the `/var/log/system.log` file to fix the missing visualizations.
+
+|Run {agent} with the System integration
+|On the `[Metrics System] Host overview` dashboard, only the processes ran by the `elastic-agent-user` user are shown in the CPU and memory usage lists.
+|Give read permission to the `elastic-agent` group for the `/var/log/system.log` file to fix the missing visualizations.
+
+|Run {agent} and access the {agent} dashboards
+|On the `[Elastic Agent] Agents info` dashboard, visualizations including `Most Active Agents` and `Integrations per Agent` are missing data.
+|Give read permission to the `elastic-agent` group for the `/var/log/system.log` file to fix the missing visualizations.
+
+|Run {agent} and access the {agent} dashboards
+|On the `[Elastic Agent] Integrations` dashboard, visualizations including `Integration Errors Table`, `Events per integration` and `Integration Errors` are missing data.
+|Give read permission to the `elastic-agent` group for the `/var/log/system.log` file to fix the missing visualizations.
 
-|<<elastic-agent-status-command,`status` command>>
-|Returns only...
-|`Example error message...`
 |===
 
 .Linux
 [options,header]
 |===
-|Command |Behavior in unprivileged mode |Error message
+|Action |Behavior in unprivileged mode |Resolution
+
+|Run {agent} with the System integration
+|Log file error: `[elastic_agent.filebeat][error] Harvester could not be started on new file: /var/log/auth.log.1, Err: error setting up harvester: Harvester setup failed. Unexpected file opening error: Failed opening /var/log/auth.log.1: open /var/log/auth.log.1: permission denied`
+|Give read permission to the `elastic-agent` group for the `/var/log/syslog` and `/var/log/auth.log` files to fix this error.
```

Comment:
I would actually recommend just adding the `elastic-agent-user` to the `adm` group on the host. This will allow the Elastic Agent to read the contents of that file without requiring changing the permissions of the file.